npm/ng-packs/packages/setting-management/config/src/lib/components/email-setting-group/email-setting-group.component.html

```diff
@@ -1,8 +1,10 @@
-<h2>{{ 'AbpSettingManagement::Menu:Emailing' | abpLocalization }}</h2>
-
-<hr class="my-2" />
-
-<form *ngIf="form" [formGroup]="form" class="py-2 abp-md-form"  (ngSubmit)="submit()" [validateOnSubmit]="true">
+<form
+  *ngIf="form"
```

Comment:
Can you please use the new control flow `@if` instead `*ngIf`

Reply:
@masumulu28 also updated npm\ng-packs\packages\feature-management\src\lib\components\feature-management-tab\feature-management-tab.component.html